cmd/prow-job-dispatcher/main.go

```diff
@@ -754,5 +785,23 @@ func main() {
 		logrus.Info("Finished dispatching and create no PR, exiting ...")
 		os.Exit(0)
 	}
-	createPR(o)
+	logrus.WithField("targetDir", o.targetDir).Info("Changing working directory ...")
```

Comment:
This part about PR creation is re-added. It was here before. Dispatcher as daemon diverged too much in PR creation and I had to re-introduce the old way for non-daemonized run.